awx/main/models/jobs.py

```diff
@@ -710,13 +710,23 @@ def notification_data(self, block=5):
                                           skipped=h.skipped,
                                           rescued=h.rescued,
                                           ignored=h.ignored)
+           
+        # get joboutput for data
+        joboutput = ""
+        try:
+            # check if information is available
+            joboutput = self._result_stdout_raw(redact_sensitive=True, escape_ascii=True)
```

Comment:
Generating raw stdout in this way has the potential to be prohibitively expensive.

Reply:
It also has a very high probability of making the notification unreliable due to the amount of data being transported over arbitrary APIs.

Reply:
Okay, I took it out for now, but I think we should find a way to at least make it available through the notifications templates  
so everyone can at least decide for himself whether or not he needs it